server.js

```diff
@@ -25,8 +25,8 @@ import debug from './src/helpers/debug';
 
 import Html from './src/helpers/Html';
 
-import { setUserAgent } from './src/redux/modules/audioplayer';
-import { setOption } from './src/redux/modules/options';
+import { setUserAgent } from './src/redux/actions/audioPlayerActions.js';
```

Comment:
thoughts about the file names just `/actions/audioplayer` Also, audioplayer has not been cased like `audioPlayer` throughout